website/docs/docs/collaborate/model-query-history.md

```diff
@@ -67,23 +67,20 @@ The model query history feature uses the credentials in your production environm
 <Lightbox src="/img/docs/collaborate/dbt-explorer/model-query-credentials.jpg" width="50%" title="Confirm your deployment credentials in your environment settings page." />
 
 4. Copy or cross reference those credential permissions with the warehouse permissions and grant your user the right permissions.
-
-    <Expandable alt_header="For Snowflake">
-
-    This feature makes use of metadata tables available to Snowflake Enterprise tier accounts or higher, `QUERY_HISTORY` and `ACCESS_HISTORY`. The Snowflake user used in the production environment must have `GOVERNANCE_VIEWER` permissions to view the data. 
-
-    This can be granted to this user by your `ACCOUNTADMIN` user in Snowflake. For more details, view the snowflake docs [here](https://docs.snowflake.com/en/sql-reference/account-usage#enabling-other-roles-to-use-schemas-in-the-snowflake-database).
-
-    </Expandable>
-
-    <Expandable alt_header="For BigQuery">
-
-    This feature uses the metadata from the `INFORMATION_SCHEMA.JOBS` view in BigQuery. To access this, the user configured for your production environment must have the following [IAM roles](https://cloud.google.com/bigquery/docs/access-control) for your BigQuery project:
-
-    - `roles/bigquery.resourceViewer`
-    - `roles/bigquery.jobs.create`
-
-    </Expandable>
+   
+   - #### For Snowflake
+     This feature makes use of metadata tables available to Snowflake Enterprise tier accounts or higher, `QUERY_HISTORY` and `ACCESS_HISTORY`. The Snowflake user used in the production environment must have `GOVERNANCE_VIEWER` permissions to view the data.
+     Before enabling Model query history, your `ACCOUNTADMIN` must run the following grant statement in Snowflake to ensure for access:
```

Comment:
Suggest slight alteration because I believe other roles can run this statement
```suggestion
     Before enabling model query history, your account admin must run the following grant statement in Snowflake to ensure for access:
```

Reply:
hey @matthewshaver , @jzhu13 has confirmed that [snowflake docs](https://docs.snowflake.com/en/sql-reference/account-usage#enabling-other-roles-to-use-schemas-in-the-snowflake-database) only mentions `access to schemas in this database can be granted by a user with the ACCOUNTADMIN role` and since it's explicitly mentioned in the snowflake doc -- we want to align with that.